lib/jekyll-gist/gist_tag.rb

```diff
@@ -35,20 +35,26 @@ def render(context)
         end
       end
 
+      def self.client
+        @client ||= Octokit::Client.new :access_token => ENV["JEKYLL_GITHUB_TOKEN"]
+      end
+
       private
 
       def determine_arguments(input)
         matched = input.match(%r!\A([\S]+|.*(?=\/).+)\s?(\S*)\Z!)
         [matched[1].strip, matched[2].strip] if matched && matched.length >= 3
       end
 
+      # rubocop:disable Style/PreferredHashMethods
```

Comment:
I'd rather avoid comments in files, it defeats the purpose of having an automatically generated todo